pkg/strategy/linregmaker/strategy.go

```diff
@@ -0,0 +1,786 @@
+package linregmaker
+
+import (
+	"context"
+	"fmt"
+	"github.com/c9s/bbgo/pkg/dynamicmetric"
+	"sync"
+
+	"github.com/c9s/bbgo/pkg/indicator"
+	"github.com/c9s/bbgo/pkg/util"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/bbgo"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// TODO: Docs
+
+const ID = "linregmaker"
+
+var notionModifier = fixedpoint.NewFromFloat(1.1)
+var two = fixedpoint.NewFromInt(2)
+
+var log = logrus.WithField("strategy", ID)
+
+func init() {
+	bbgo.RegisterStrategy(ID, &Strategy{})
+}
+
+type Strategy struct {
+	Environment          *bbgo.Environment
+	StandardIndicatorSet *bbgo.StandardIndicatorSet
+	Market               types.Market
+	ctx                  context.Context
```

Comment:
put private fields to the end of this struct

Reply:
done